res/controllers/Traktor-Kontrol-S4-MK2-hid-scripts.js

```diff
@@ -1138,6 +1175,36 @@ TraktorS4MK2.callbackLoopMove = function(field) {
   }
 }
 
+TraktorS4MK2.sendLoopSizeMessage = function(group, beats) {
+  var LoopSizeIndicators = {
+      "deck2" : [0x2B, 0x3A],
+      "deck1" : [0x1B, 0x2A],
+  }
+  start_signal = LoopSizeIndicators[group]
+
+  var sizesArray = {
+    64: [1,2,4,5,6,7,9,10,11,13],
```

Comment:
Are these arrays for lighting specific segments of the LED display?  I'd prefer a general function that can display arbitrary integers for easier reading/testing and possible future reuse.